infrastructure/src/main/java/org/corfudb/infrastructure/NettyServerRouter.java

```diff
@@ -95,12 +109,24 @@ public void setServerContext(ServerContext serverContext) {
      * @param inMsg  Incoming message to respond to.
      * @param outMsg Outgoing message.
      */
+    @Deprecated
     public void sendResponse(ChannelHandlerContext ctx, CorfuMsg inMsg, CorfuMsg outMsg) {
         outMsg.copyBaseFields(inMsg);
         ctx.writeAndFlush(outMsg, ctx.voidPromise());
         log.trace("Sent response: {}", outMsg);
     }
 
+    /**
+     * Send a response message through this router.
+     *
+     * @param response The response message to send.
+     * @param ctx      The context of the channel handler.
+     */
+    public void sendResponse(ResponseMsg response, ChannelHandlerContext ctx) {
+        ctx.writeAndFlush(response, ctx.voidPromise());
+        log.trace("Sent response: {}", response);
```

Comment:
can you wrap this with `if (log.isTraceEnable)`

Reply:
Done.